vlasovsolver/cuda_acc_map_kernel.cuh

```diff
@@ -11,7 +11,7 @@ struct Column
    int i,j;                                       // Blocks' perpendicular coordinates
 };
 
-extern Realf* acceleration_1_wrapperCaller
+extern void acceleration_1_wrapperCaller
```

Comment:
As an addendum to the comment above: do we *really* need the wrapper, the wrapperCaller and the kernel as separate functions? I understand they have grown organically, and it works like this... but ugh.

Reply:
Probably the wrappercaller can indeed be removed, I'll give it a go. The wrapper itself of course is needed, especially when we improve parallelization and have multiple different kernels.